### ISSUE TITLE
PyCharm OCL run configuration 

### ISSUE THREAD

**Issue by stfanbrg** [ScribesZone/ScribesTools#4]
Lors de la configuration de Pycharm il est proposé de configurer un run  
http://scribestools.readthedocs.org/en/latest/useocl/index.html#creating-a-run-configuration

je ne trouve pas le répertoire .idea/runConfigurations 
Faut-il le créer ou il manque une étape lors de l'installation de Pycharm ?

**Comment by escribis** [comment]
Yes, the directory ``runConfigurations`` have to be created if it is not there. This directory is created only when one want to share "run configurations".

```
Copy the (checkScn001Razamanaz.xml) configuration in the directory .idea/runConfigurations of your project (create the directory runConfigurations if it does not exist already). 'use
``` 

What is missing in the doc, is the way to get the 'use-check.sh' script. You can write your own version (it is just launching use tool) or make    a git clone of the repository ScribesZone/ScribesInfra at the SAME LEVEL of your group repository (NOT INSIDE). This will be explained later as there will be more tools there.

I'm figthing to get solutions that work on all platforms but that's not so easy. Investigating as well pycharm server settings, but not sure if this will be easy to use it with firewall, no firewall, multi-platform, etc. So at the moment creating a PyCharm configuration is not direct.

Hope this help. Not sure as the answer was already in the doc, it's possible that I do not understand the question. Just let me know.

**Comment by stfanbrg** [comment]
Thanks for the answer. So, the ./idea didn't exist. May be is not normal. I wil check the doc

**Comment by jensMF** [comment]
The ScribesInfra repository is empty, where can I get the 'use-check.sh' script now?

**Comment by ArmelZy** [comment]
Please, No solution to this issue in 2021?
